addons/emp/functions/fnc_fireEMP.sqf

```diff
@@ -16,18 +16,17 @@ if (!isServer) exitWith {};
 private _pos = ASLToAGL _pos;
 
 // if unit is null, we spawn something and use as EMP on the given position
-if (isNull _object && _spawnDevice) then 
+if (isNull _object && _spawnDevice) then
 {
 	_object = createVehicle ["Crows_Emp_Device", _pos, [], 0, "CAN_COLLIDE"];
 
 	_pos = ASLToAGL getPosASL _object;
 
-	// set zeus editable 
+	// set zeus editable
 	["zen_common_addObjects", [[_object], objNull]] call CBA_fnc_serverEvent;
 };
 
-// create visual effects for all
-[[_pos, _range],QPATHTOF(functions\fnc_playerEffect.sqf)] remoteExec ["execVM", [0,-2] select isDedicated];
+[QGVAR(playerEffect), [_pos, _range, [] call CBA_fnc_currentUnit]] call CBA_fnc_globalEvent;
```

Comment:
Any particular reason using ``[] call CBA_fnc_currentUnit`` here? If this is executed on the server, then it would give the server, and not the player receiving this event? 
Or am I missing something? 

I do agree that using it over ``player`` is probably better on the ``fnc_playerEffect`` side, as it will take zeus RC into account.

Reply:
Right that is called from server, so yeah has to just be done in each handler. I'll fix that quickly